packages/scripts/config/webpack.config.js

```diff
@@ -67,12 +92,51 @@ const config = {
 				test: /\.svg$/,
 				use: [ '@svgr/webpack', 'url-loader' ],
 			},
+			{
+				test: /\.(sc|sa|c)ss$/,
```

Comment:
Note to myself. This is how @ItsJonQ implemented CSS support for Storybook in Gutenberg:

https://github.com/WordPress/gutenberg/blob/1c7d80b7ad8025b2cead7866e49103911cf4889a/storybook/webpack.config.js#L11-L31

The order seems different but we can fix that. The only thing I need to understand better is what `MiniCssExtractPlugin` plays here.

Reply:
This is the last Gutenberg config that had CSS support embeded in the webpack config:

https://github.com/WordPress/gutenberg/blob/29370a503d4d59e4b64530195c066fc2c3804061/webpack.config.js